lib/travis/build/addons/rethinkdb.rb

```diff
@@ -0,0 +1,44 @@
+require 'shellwords'
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class Rethinkdb < Base
+        SUPER_USER_SAFE = false
```

Comment:
Since everything that requires `sudo` happens before `sudo` is disabled, you can enable this on containers:

```ruby
SUPER_USER_SAFE = true
```